test/onnx/gen_onnx.py

```diff
@@ -1315,6 +1315,22 @@ def conv_bias_test():
     return ([node], [x, y, z], [out])
 
 
+@onnx_test()
```

Comment:
Could you please also add a test to verify the calculations, with these biases. These newly added tests check the expected graph, I would recommend we should also verify the numerical calculations.

Reply:
Yup understood

Reply:
There are tests in the onnx_backend_test that should cover this

seeing test_convInteger + test+convinteger_without_padding in onnx_backend_test.py 

I can add a verify test with the pad though to ensure this is fine